red5pro-installer/r5p_config_coturn.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+############################################################################################################
+# Red5 Pro Coturn configuration for single Red5 Pro server and Red5 Pro Stream Manager                     #
+############################################################################################################
+
+# COTURN_ENABLE="true"
+# COTURN_ADDRESS="stun:1.2.3.4:3478"
+
+RED5_HOME="/usr/local/red5pro"
+
+log_i() {
+    log
+    printf "\033[0;32m [INFO]  --- %s \033[0m\n" "${@}"
+}
+log_d() {
+    log
+    printf "\033[0;33m [INFO]  --- %s \033[0m\n" "${@}"
+}
+log_w() {
+    log
+    printf "\033[0;35m [WARN] --- %s \033[0m\n" "${@}"
+}
+log_e() {
+    log
+    printf "\033[0;31m [ERROR]  --- %s \033[0m\n" "${@}"
+}
+log() {
+    echo -n "[$(date '+%Y-%m-%d %H:%M:%S')]"
+}
+
+config_red5pro_coturn(){
+
+    if [[ "$COTURN_ENABLE" == "true" ]]; then
+        log_i "Red5Pro Coturn configuration - enable"
+    
+        if [ -z "$COTURN_ADDRESS" ]; then
+            log_e "Parameter COTURN_ADDRESS is empty. EXIT."
+            exit 1
+        fi
+
+        log_i "Config Coturn address: $RED5_HOME/conf/network.properties"
+        stun_address='stun.address=.*'
+        stun_address_new="stun.address=${NODE_CLOUDSTORAGE_AWS_BUCKET_ACL_POLICY}"
```

Comment:
Hi @iolesyk, Update the variable from `NODE_CLOUDSTORAGE_AWS_BUCKET_ACL_POLICY` to `COTURN_ADDRESS`